### PR TITLE
Fix typo in ban docs

### PR DESCRIPTION
**Pull request by pe8ter**
#### PR checklist

- [ ] Addresses an existing issue: #0000
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [x] Documentation update

#### Overview of change:

Just a small typo.

#### CHANGELOG.md entry:

[docs] Fix docs typo

### CODE REVIEW

**Comment by pe8ter** [comment]
I'm not sure where all that auto-formatting came from. I'll tamp that down and force push a change.

**Comment by pe8ter** [comment]
I'm at a loss here. It looks like the pre-commit hook is formatting my code, but CircleCI fails on the linting step for the changes the pre-commit hook made. How do I proceed?

File: src/rules/banRule.ts [hunk 1]
```diff
@@ -133,10 +137,14 @@ function parseOptions(args: Array<string | string[] | OptionsInput>): Options {
...
                 case 1:
-                    functions.push({name: arg.name[0], message: arg.message});
+                    functions.push({ name: arg.name[0], message: arg.message });
                     break;
                 default:
-                    methods.push({name: arg.name[arg.name.length - 1], object: arg.name.slice(0, -1), message: arg.message});
+                    methods.push({
+                        name: arg.name[arg.name.length - 1],
+                        object: arg.name.slice(0, -1),
+                        message: arg.message
```
Comment (johnwiseheart) [inline src/rules/banRule.ts]: So just to confirm, if you move `message` to the beginning of this object, the commit hook moves it back to the end?

Comment (pe8ter) [reply src/rules/banRule.ts]: It doesn't and I see why. The formatter on my commit changes the line

```typescript
methods.push({name: arg.name[arg.name.length - 1], object: arg.name.slice(0, -1), message: arg.message});
```

into the snippet you posted.

If I then make another commit, the formatter runs again and move `message` up into the correct sorting position. It looks like the formatter can't make both edits in one commit.

Comment (pe8ter) [reply src/rules/banRule.ts]: What I can do is squash this second commit down and try pushing up again.

Comment (pe8ter) [reply src/rules/banRule.ts]: The formatter is not able to correct this file no matter how many times it runs.

Comment (pe8ter) [reply src/rules/banRule.ts]: I manually moved the `message` line up and CI passes now. I guess I don't understand how much the formatter is supposed to change and falsely believed it should correct the sort order.

File: src/rules/banRule.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ export class Rule extends Lint.Rules.AbstractRule {
               * the name of the function in an array with one element: \`["functionName"]\`
               * an object in the following format: \`{"name": "functionName", "message": "optional explanation message"}\`
             * banning methods:
-              * an array with the object name, method name and optional message: \`["functionName", "methodName", "optional message"]\`
+              * an array with the object name, method name and optional message: \`["objectName", "methodName", "optional message"]\`
```
Comment (pe8ter) [inline src/rules/banRule.ts]: This is the only change that I explicitly made for this PR. The rest of the changes are auto-formatter changes.

File: src/rules/banRule.ts (hunk 1 above)
Comment (johnwiseheart) [reply src/rules/banRule.ts]: Awesome. The autoformatter is a bit of a pain, but hopefully will let us work towards a codebase that is nicer without ruining everyones open PRs.

**Comment by pe8ter** [comment]
Thanks!

**Pull request merged**